packages/core/src/families/evm/types.ts

```diff
@@ -0,0 +1,38 @@
+import type { z } from "zod";
+import type BigNumber from "bignumber.js";
+import type { TransactionCommon } from "..";
+import type {
+  schemaRawEvmTransaction,
+  schemaRawEvmTransactionEIP1559,
+} from "./validation";
+
+export interface EvmTransactionBase extends TransactionCommon {
```

Comment:
Do we need to support `mode` field ? I don't see in our ethereum family. @IAmMorrow